api/v3/Activity/Getactionlinks.php

```diff
@@ -1,51 +1,65 @@
 <?php
 
+/**
+ * @file
+ * Activity.getactionlinks file.
```

Comment:
Again. What extra info this `Activity.getactionlinks file.` text adds to the documentation that the file name doesn't have.
What's the need of having it at the first place.